apps/datahub-e2e/src/e2e/datasetDetailPage.cy.ts

```diff
@@ -641,14 +641,16 @@ describe('api form', () => {
     cy.get('@secondInput').clear()
     cy.get('@secondInput').type('87')
 
-    cy.get('@apiForm').find('gn-ui-dropdown-selector').click()
-    cy.get('button[data-cy-value="csv"]').click()
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(3000)
```

Comment:
Instead of adding this wait time, it might be better to intercept & mock the results from the test API. But this can be done in a followup PR